oak/proto/application.proto

```diff
@@ -94,6 +94,11 @@ message GrpcClientConfiguration {
   // The endpoint address of the external gRPC service.
   // `address` is represented as an "ip_address:tcp_port" string.
   string address = 1;
+  // Domain name of the external gRPC service.
+  string domain_name = 2;
```

Comment:
Is this in addition to the one from the address field? Does this override that? I'm a bit confused.